fcsparser/api.py

```diff
@@ -424,7 +447,7 @@ def reformat_meta(self):
         meta['_channel_names_'] = self.get_channel_names()
 
 
-def parse(path, meta_data_only=False, output_format='DataFrame', compensate=False,
+def parse(path, dataset_start=0, meta_data_only=False, output_format='DataFrame', compensate=False,
```

Comment:
let's stick this parameter at the end to not surprise folks that have code written without keyword arguments provided explicitly. 

i.e.,

imagine someone wrote the code:

```
parse(path, False)
```